packages/renderer/src/App.svelte

```diff
@@ -318,6 +319,9 @@ window.events?.receive('navigate', (navigationRequest: unknown) => {
             navigationHint="details">
             <RouteDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
           </Route>
+          <Route path="/kubernetes/port-forward" breadcrumb="Port Forwarding" navigationHint="root">
```

Comment:
in k8s section we follow camelcase, but I guess this doesn't matter (I see other routes are using dashes)

So should be portForward, this this is just a nitpick